packages/components/src/IconList/src/IconList.module.css

```diff
@@ -0,0 +1,8 @@
+.hop-iconList  {
+    --hop-icon-list-component-gap: var(--hop-space-inline-xs);
```

Comment:
Wouldn't it be simpler to name the variable `--hop-iconList-gap` with the `component` implicit?

Reply:
100%, wasn't sure about the format we wanted. I'll make the changes

Reply:
I'll write in slack, but i'll do those changes in another Pr in order to speed things up a bit